grails-app/services/org/pih/warehouse/order/OrderService.groovy

```diff
@@ -572,14 +627,28 @@ class OrderService {
      * @param inputStream
      * @return
      */
-    List parseOrderItems(String text) {
+    List parseOrderItems(InputStream inputStream) {
```

Comment:
Why did you change the signature here? Why not just use the following?
```
orderService.parseOrderItems(multipartFile.inputStream.text)
```

Reply:
I think I had some kind of issues here, but I'll check  again